defaults.go

```diff
@@ -68,6 +68,16 @@ func (dns *Msg) SetRcodeFormatError(request *Msg) *Msg {
 	return dns
 }
 
+// SetRcodeNotImplemented creates a message with NotImplemented set.
+func (dns *Msg) SetRcodeNotImplemented(request *Msg) *Msg {
```

Comment:
Probably not useful at this stage (usefulness of SetRcodeFormatError is also questionable)